shepherd_bms/src/analyzer.cpp

```diff
@@ -250,13 +251,13 @@ void Analyzer::disableTherms()
 
     for(uint8_t c = 1; c < NUM_CHIPS; c+= 2)
```

Comment:
```suggestion
    for(uint8_t c = 1; c < NUM_CHIPS; c++)
```

Reply:
I think you meant to do this too?

Reply:
i wasnt totally sure, looking at the math done on c in that loop. right now +=2 results in it disabling chips 0-3, which is different than that chip skipping logic earlier. Not sure if it falls under same category and needs to be changed or if it serves a different purpose

Reply:
How does it disable 0-3? Doesn't it just disable evens? Either way I think this is the same case and should hit every chip

Reply:
THERM_DISABLE[(c - 1) / 2][therm - START_THERM])

so will disable 1-1 /2 = 0,  3-1 /2 = 1,  5-1 /2 = 2, 7-1 /2 = 3. But yeah i can change to just do every chip then